AppCreationScripts/apps.json

```diff
@@ -36,11 +36,12 @@
         "settingFile": "/JavaScriptSPA/config.js", 
             "replaceTokens": 
               {
-            "clientId": "Enter_the_Application_Id_Here",
-            "authority": "Enter_the_Cloud_Instance_Here/Enter_the_Tenant_Info_Here",
-            "redirectUri": "Enter_the_Redirect_Uri_Here",
-            "graphMeEndpoint": "Enter_the_Graph_Endpoint_Here/v1.0/me",
-            "graphMailEndpoint": "Enter_the_Graph_Endpoint_Here/v1.0/me/messages"
+              "appId": "Enter_the_Application_Id_Here",
+              "tenantId": "Enter_the_Tenant_Info_Here",
+              "authorityEndpointHost": "Enter_the_Cloud_Instance_Id_Here",
+              "redirectUri": "Enter_the_Redirect_Uri_Here",
+              "msgraphEndpointHost": "Enter_the_Graph_Endpoint_Here/v1.0/me",
```

Comment:
"msgraphEndpointHost":"Enter_the_Graph_Endpoint_Here" is enough. The entire string is replaced with host url(protocol+domain+trailing slash and you would want to keep "v1.0/me".

Reply:
sorry, that was the outdated file. recent pushes fixed this.